README.md

```diff
@@ -6,7 +6,13 @@
 [![SonarCloud Coverage](https://img.shields.io/sonar/coverage/michelin_kstreamplify?logo=sonarcloud&server=https%3A%2F%2Fsonarcloud.io&style=for-the-badge)](https://sonarcloud.io/component_measures?id=michelin_kstreamplify&metric=coverage&view=list)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg?logo=apache&style=for-the-badge)](https://opensource.org/licenses/Apache-2.0)
 
-Kstreamplify is a Java library that brings new features on top of Kafka Streams.
+So here you are, sitting on your chair  in that open space, scratching your head thinking "I really wish Kafka Streams was easier to implement... 
```

Comment:
I think a more serious and concise introduction will be more suitable for a package, but other opinions are welcome 😄

Reply:
Agree with that, even if most others people validate this one, it can be better if the intro is a bit more serious 😁